### ISSUE TITLE
换MySQL的时候出错日志如下。

### ISSUE THREAD

**Issue by LuChengX** [shadowsocks/shadowsocks-manager#214]
[2017-05-23 04:09:24.842] [ERROR] system - { Error: ER_BAD_FIELD_ERROR: Unknown column 'sumFlow' in 'field list'
    at Query.Sequence._packetToError (/root/ssm/node_modules/mysql/lib/protocol/sequences/Sequence.js:52:14)
    at Query.ErrorPacket (/root/ssm/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/root/ssm/node_modules/mysql/lib/protocol/Protocol.js:280:23)
    at Parser.write (/root/ssm/node_modules/mysql/lib/protocol/Parser.js:75:12)
    at Protocol.write (/root/ssm/node_modules/mysql/lib/protocol/Protocol.js:39:16)
    at Socket.<anonymous> (/root/ssm/node_modules/mysql/lib/Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:547:20)
    --------------------
    at Protocol._enqueue (/root/ssm/node_modules/mysql/lib/protocol/Protocol.js:141:48)
    at Connection.query (/root/ssm/node_modules/mysql/lib/Connection.js:208:25)
    at /root/ssm/node_modules/knex/lib/dialects/mysql/index.js:152:18
    at Promise._execute (/root/ssm/node_modules/bluebird/js/release/debuggability.js:300:9)
    at Promise._resolveFromExecutor (/root/ssm/node_modules/bluebird/js/release/promise.js:483:18)
    at new Promise (/root/ssm/node_modules/bluebird/js/release/promise.js:79:10)
    at Client_MySQL._query (/root/ssm/node_modules/knex/lib/dialects/mysql/index.js:146:12)
    at Client_MySQL.query (/root/ssm/node_modules/knex/lib/client.js:197:17)
    at Runner.<anonymous> (/root/ssm/node_modules/knex/lib/runner.js:146:36)
    at Runner.tryCatcher (/root/ssm/node_modules/bluebird/js/release/util.js:16:23)
    at Runner.query (/root/ssm/node_modules/bluebird/js/release/method.js:15:34)
    at /root/ssm/node_modules/knex/lib/runner.js:65:21
    at tryCatcher (/root/ssm/node_modules/bluebird/js/release/util.js:16:23)
    at /root/ssm/node_modules/bluebird/js/release/using.js:185:26
    at tryCatcher (/root/ssm/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/root/ssm/node_modules/bluebird/js/release/promise.js:512:31)
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlState: '42S22',
  index: 0 }

本地MySQL和远程MySQL都是这样的问题。QAQ

**Comment by gyteng** [comment]
请提供更详细的信息

**Comment by whao** [comment]
@gyteng 我也是同样的问题
```
Linux Weis-MSI-GT60-VM-Ubuntu 4.8.0-52-generic #55~16.04.1-Ubuntu SMP Fri Apr 28 14:36:29 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
mysql  Ver 15.1 Distrib 10.1.23-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
shadowsocks-manager 0.6.24
```
default.yml
```
type: s
empty: false

shadowsocks:
  address: 127.0.0.1:6001

manager:
  address: 127.0.0.1:6002
  password: '123456'

#db: 'ss.sqlite'

db:
  host: '127.0.0.1'
  user: 'ssmgr'
  password: 'ssmgr'
  database: 'ssmgr'
```
webgui.yml
```
type: m
empty: false

manager:
  address: 127.0.0.1:6002
  password: '123456'
  # 这部分的端口和密码需要跟上一步 manager 参数里的保持一致
plugins:
  flowSaver:
    use: true
  user:
    use: true
  account:
    use: true
    pay:
      hour:
        price: 0.03
        flow: 500000000
      day:
        price: 0.5
        flow: 7000000000
      week:
        price: 3
        flow: 50000000000
      month:
        price: 10
        flow: 200000000000
      season:
        price: 30
        flow: 200000000000
      year:
        price: 120
        flow: 200000000000
  email:
    use: true
    username: 'username'
    password: 'password'
    host: 'smtp.your-email.com'
    # 这部分的邮箱和密码是用于发送注册验证邮件，重置密码邮件使用的
  webgui:
    use: true
    host: '0.0.0.0'
    port: '8080'
    site: 'http://yourwebsite.com'
    gcmSenderId: '456102641793'
    gcmAPIKey: 'AAAAGzzdqrE:XXXXXXXXXXXXXX'
  alipay:
    use: true
    appid: 2015012108272442
    notifyUrl: ''
    merchantPrivateKey: 'xxxxxxxxxxxx'
    alipayPublicKey: 'xxxxxxxxxxx'
    gatewayUrl: 'https://openapi.alipay.com/gateway.do'

#db: 'webgui.sqlite'

db:
  host: '127.0.0.1'
  user: 'ssmgr'
  password: 'ssmgr'
  database: 'ssmgr_webgui'
```
所有都运行在本地。在开启`ssmgr -c ~/.ssmgr/webgui.yml`后`ssmgr -c ~/.ssmgr/default.yml`会报如下错误
```
$ ssmgr -c ~/.ssmgr/default.yml
/usr/lib/node_modules/shadowsocks-manager/lib/
[2017-05-24 02:03:21.913] [INFO] system - System start.
[2017-05-24 02:03:51.629] [ERROR] system - { Error: ER_BAD_FIELD_ERROR: Unknown column 'sumFlow' in 'field list'
    at Query.Sequence._packetToError (/usr/lib/node_modules/shadowsocks-manager/node_modules/mysql/lib/protocol/sequences/Sequence.js:52:14)
    at Query.ErrorPacket (/usr/lib/node_modules/shadowsocks-manager/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/usr/lib/node_modules/shadowsocks-manager/node_modules/mysql/lib/protocol/Protocol.js:280:23)
    at Parser.write (/usr/lib/node_modules/shadowsocks-manager/node_modules/mysql/lib/protocol/Parser.js:75:12)
    at Protocol.write (/usr/lib/node_modules/shadowsocks-manager/node_modules/mysql/lib/protocol/Protocol.js:39:16)
    at Socket.<anonymous> (/usr/lib/node_modules/shadowsocks-manager/node_modules/mysql/lib/Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at TCP.onread (net.js:561:20)
    --------------------
    at Protocol._enqueue (/usr/lib/node_modules/shadowsocks-manager/node_modules/mysql/lib/protocol/Protocol.js:141:48)
    at Connection.query (/usr/lib/node_modules/shadowsocks-manager/node_modules/mysql/lib/Connection.js:208:25)
    at /usr/lib/node_modules/shadowsocks-manager/node_modules/knex/lib/dialects/mysql/index.js:152:18
    at Promise._execute (/usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/debuggability.js:300:9)
    at Promise._resolveFromExecutor (/usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/promise.js:483:18)
    at new Promise (/usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/promise.js:79:10)
    at Client_MySQL._query (/usr/lib/node_modules/shadowsocks-manager/node_modules/knex/lib/dialects/mysql/index.js:146:12)
    at Client_MySQL.query (/usr/lib/node_modules/shadowsocks-manager/node_modules/knex/lib/client.js:197:17)
    at Runner.<anonymous> (/usr/lib/node_modules/shadowsocks-manager/node_modules/knex/lib/runner.js:146:36)
    at Runner.tryCatcher (/usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/util.js:16:23)
    at Runner.query (/usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/method.js:15:34)
    at /usr/lib/node_modules/shadowsocks-manager/node_modules/knex/lib/runner.js:65:21
    at tryCatcher (/usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/util.js:16:23)
    at /usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/using.js:185:26
    at tryCatcher (/usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/lib/node_modules/shadowsocks-manager/node_modules/bluebird/js/release/promise.js:512:31)
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlState: '42S22',
  index: 0 }
```
回到数据库检查可以两个数据库已经创建了该有的表了
```
MariaDB [ssmgr]> show tables;
+-----------------+
| Tables_in_ssmgr |
+-----------------+
| account         |
| command         |
| flow            |
+-----------------+

MariaDB [ssmgr_webgui]> show tables;
+------------------------+
| Tables_in_ssmgr_webgui |
+------------------------+
| account                |
| account_plugin         |
| alipay                 |
| command                |
| email                  |
| flow                   |
| notice                 |
| push                   |
| saveFlow               |
| saveFlow5min           |
| saveFlowDay            |
| saveFlowHour           |
| server                 |
| user                   |
| webguiSetting          |
+------------------------+
```
报错里面的`sumFlow`没有看见，不知道是怎么回事，目前情况就是这样，有想到的我再来补充。
谢谢：）

**Comment by gyteng** [comment]
请给出MySQL版本，node版本，ssmgr版本等一些详细信息，以及重现错误的方法

**Comment by whao** [comment]
开头不就给了嘛
```
$ node -v
v7.10.0
$ sudo npm -v
4.6.1
```
重现看这里: [ogv](https://drive.google.com/open?id=0B2Q23IWw5Q-kNTJCeDRIYUVJWlU)

**Comment by gyteng** [comment]
type s 端没有必要用mysql，建议先用sqlite，而且后面还会把sqlite去掉

**Comment by Jarvanh** [comment]
同样的问题，换成sqlite就好了。

**Comment by gyteng** [comment]
fix in #6403fa6

**Comment by whao** [comment]
@gyteng 原来在shadowsocks js里面 找半天没找到。大佬爱你么么哒
